src/custom/pages/About/About.styled.tsx

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components'
+import AppBody from 'pages/AppBody'
+
+export const AppBodyMod = styled(AppBody)`
+  padding: 0 24px 24px;
+  max-width: ${({ theme }) => theme.appBody.maxWidth.content};
+`
+
+export const Title = styled.h1`
+  font-size: 32px;
+`
+
+export const Content = styled.div`
+  font-size: 15px;
+  margin: 0 0 28px;
+  display: block;
+
+  > h2 {
+    font-size: 18px;
+  }
+
+  > h2:not(:first-of-type)::before {
```

Comment:
- Markdown `##` titles (H2) get a `1px border` applied using a CSS pseudo element (except the first H2).